### PR TITLE
Sonchau/moh part 9

### PR DESCRIPTION
**Pull request by SonQBChau**
- add convert scripts
- update readme with instructions on how to create synthetic data

### CODE REVIEW

File: chord_metadata_service/mohpackets/data/README.md [hunk 1]
```diff
@@ -1,7 +1,49 @@
 # About
 
-This folder contains synthetic data for MoH models.
+This folder contains synthetic data for **MoH models**.
 
-The **schemas** contains the blueprints to generate the data using [Mockaroo](https://www.mockaroo.com/) ("RESTORE FROM BACKUP...")
+The **schemas** contains the blueprints to generate the data.
 
-The **fixtures** contains the synthetic data in JSON format for quick import into PostgreSQL.
+The **synthetic_data** folder contains the generated data.
```
Comment (daisieh) [inline chord_metadata_service/mohpackets/data/README.md:7]: There is currently data in the synthetic_data folder in the repo: is that an already-created instance of this test data?

**Review by daisieh — APPROVED** [review]
This looks great. Will I be able to use this synthetic data for https://candig.atlassian.net/browse/DIG-1008?

File: chord_metadata_service/mohpackets/data/README.md (hunk 1 above)
Comment (SonQBChau) [reply chord_metadata_service/mohpackets/data/README.md:7]: yes, I included the ready-made data in the folder to make it easy for everyone, so we can just ingest it.

**Comment by SonQBChau** [comment]
> This looks great. Will I be able to use this synthetic data for https://candig.atlassian.net/browse/DIG-1008?

Yes, I have to think about the details, but this is the goal, make it easy to have mock data to test.

**Pull request merged**